packages/ppx_defer/ppx_defer.0.3.0/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Hezekiah M. Carty <hez@0ok.org>"
+authors: [ "Hezekiah M. Carty <hez@0ok.org>" ]
+license: "MIT"
+homepage: "https://github.com/hcarty/ppx_defer"
+bug-reports: "https://github.com/hcarty/ppx_defer/issues"
+dev-repo: "https://github.com/hcarty/ppx_defer.git"
+build:[ "jbuilder" "build" "-p" name "-j" jobs ]
+depends: [
+  "jbuilder" {build & >= "1.0.0+beta9"}
```

Comment:
Thanks - I'll fix And submit a separate PR to make sure the project repo is correct too.